Miguel Tuñón 8-929-2198/imprin.c

```diff
@@ -0,0 +1,18 @@
+#include<stdio.h>
+void main()
+{
+	int n,f,i=0,c;
+	
+	printf("Ingrese un valor inicial\n");
+	scanf("%d",&n);
+	printf("Ingrese un valor final\n");
+	scanf("%d",&f);
+	printf("Ingrese el contador\n");
+	scanf("%d",&i);
+	
+	for(c=n;c<=f;c+i)

```

Comment:
for(c=n;c<=f;c=c+i)

Reply:
en el for puedes colocar la variable de incremento. Como no lo hiciste en el for, el incremento ocurre despues de que te has pasado del valor final

Reply:
![32156684-e507e44e-bd0c-11e7-9522-ecab236f54c1](https://user-images.githubusercontent.com/31838625/32157106-502fd310-bd0f-11e7-81bc-848abbecba94.png)